src/pages/CommandIndex/ExecuteButton.tsx

```diff
@@ -1,27 +1,41 @@
 import { Button } from '@mui/material'
+import { JobRequestCreationContext } from 'components/JobRequestCreation'
 import { Link as RouterLink } from 'react-router-dom'
-import { AugmentedCommand } from 'types/custom-types'
+import { AugmentedCommand, StrippedSystem } from 'types/custom-types'
 
-const ExecuteButton = (command: AugmentedCommand) => {
+const ExecuteButton = (system: StrippedSystem, command: AugmentedCommand) => {
   const { namespace, systemName, systemVersion, name } = command
 
   return (
-    <Button
-      component={RouterLink}
-      to={[
-        '/systems',
-        namespace,
-        systemName,
-        systemVersion,
-        'commands',
-        name,
-      ].join('/')}
-      size="small"
-      variant="contained"
-      color="primary"
-    >
-      Execute
-    </Button>
+    <JobRequestCreationContext.Consumer>
```

Comment:
Here is where the button uses the context to update the "shared" variables before navigating to the request creation page.